environment.yml

```diff
@@ -4,30 +4,30 @@ channels:
 dependencies:
```

Comment:
Due to library inconsistencies I removed most of the set versions. We can try to find an updated set that works on our different platforms.